addOns/authhelper/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Document custom steps for Browser Based Authentication.
 - Document Authentication Report diagnostics data.
 - Sanitized post data to auth diagnostics.
+- Help content for configuration and use of Header Based Session Management via ZAP's API.
```

Comment:
I'd suggest keeping using "ZAP API" (there are only three occurrences of "ZAP's API").

Reply:
There's one in the help.